index.js

```diff
@@ -13,7 +13,7 @@ else if (process.platform === 'linux') {
 }
 
 // say stuff, speak
-exports.speak = function(voice, text, callback) {
+exports.speak = function(voice, text, callback, speed) {
```

Comment:
We do need to have callback as the final argument as that's the Node convention. Utilities such as `Bluebird.promisifyAll()` rely on it being the final argument.